hawkular-wildfly-agent/src/test/resources/org/hawkular/agent/monitor/extension/subsystem.xml

```diff
@@ -16,9 +16,9 @@
     limitations under the License.
 
 -->
+<!-- default value omitted: apiJndiName="java:global/hawkular/agent/api" -->
```

Comment:
don't even worry about commenting this here. The user will rarely, if ever, change that default attribute value - in fact, they probably don't even need to know it exists.